Dockerfile

```diff
@@ -1,21 +1,32 @@
-FROM node:latest
+FROM hypriot/rpi-node:latest
 
 ENV NODE_ENV production
-ENV MM_PORT 8080
+EXPOSE 8080
 
-WORKDIR /opt/magic_mirror
+RUN apt-get update
+
+# needed for running:
+RUN apt-get -qy install dos2unix libgtk2.0-0 libx11-xcb-dev libxtst6 libxss1 libgconf-2-4 libnss3-dev libasound2
 
+# arp-scan only needed for the module MMM-NetworkScanner, nano as editor:
+RUN apt-get -qy install nano arp-scan
+
+WORKDIR /opt/magic_mirror
+# getting MagicMirror from github
 RUN git clone --depth 1 -b master https://github.com/MichMich/MagicMirror.git .
 
+# install (without param the mirror remains black):
+RUN npm install --unsafe-perm
+
+# copy sample to real config:
+RUN cp config/config.js.sample config/config.js
+
+# create copies for unmount:
 RUN cp -R modules /opt/magic_mirror/unmount_modules
 RUN cp -R config /opt/magic_mirror/unmount_config
-RUN npm install --unsafe-perm
 
+# copy startscript into container:
 COPY docker-entrypoint.sh /opt/magic_mirror
-RUN apt-get update \
-  && apt-get -qy install dos2unix \
-  && dos2unix docker-entrypoint.sh \
-  && chmod +x docker-entrypoint.sh
+RUN dos2unix docker-entrypoint.sh && chmod +x docker-entrypoint.sh
 
-EXPOSE $MM_PORT
-ENTRYPOINT ["/opt/magic_mirror/docker-entrypoint.sh"]
+ENTRYPOINT ["/opt/magic_mirror/docker-entry-raspberry.sh"]
```

Comment:
yes, the docker-entry-raspberry.sh doen't exist anymore.

The entrypoint is set in the 2 docker-compose.yml files which overrides the default entrypoint defined in the Dockerfile ...